playbooks/roles/certbot/tasks/main.yml

```diff
@@ -29,14 +29,21 @@
     port: 80
     proto: tcp
 
-- name: add certbot PPA
-  apt_repository:
-    repo: ppa:certbot/certbot
+- name: Remove certbot apt package if installed
+  apt:
+    name: "{{ item }}"
+    state: absent
+  with_items:
+    - python-certbot-nginx
+    - certbot
 
-- name: install certbot apt package
+- name: Install snapd
   apt:
-    update_cache: yes
-    name: python-certbot-nginx
+    name: snapd
+
+# TODO: We should switch to the snap ansible module after we upgrade to Ansible 2.8 or newer.
+- name: Install certbot snap
+  command: snap install certbot --classic
```

Comment:
@mtyaka, can we add a comment to explain why we are removing the apt package and installing the snap package?

Reply:
Resolved [here])https://github.com/open-craft/ansible-playbooks/pull/181/files#diff-a664225e5a176729b75a4704ef78d6a6R32)